cmd/kubeadm/app/constants/constants_unix.go

```diff
@@ -1,7 +1,7 @@
 // +build !windows
 
 /*
-Copyright 2019 The Kubernetes Authors.
+Copyright 2020 The Kubernetes Authors.
```

Comment:
sorry if wasn't clear, the copyright for old files should not change. we should use 2020 for newly created ones.